tests/string/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -57,15 +57,17 @@ $string = "This is the value of the var named by the return value of getName():
 // $string = "This is the value of the var named by the return value of \\$object->getName(): {\${$object->getName()}}";
 $string = "I'd like an {\${beers::softdrink}}\\n";
 $string = "I'd like an {\${beers::$ale}}\\n";
+$test = "\\\\";
+$test = '\\\\';
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 <?php
 $string = 'this is a simple string';
 // $string = 'You can also have embedded\\n newlines in
 // strings this way as it is
 // okay to do';
 $string = 'Arnold once said: "I\\'ll be back"';
-$string = 'You deleted C:\\*.*?';
-$string = 'You deleted C:\\*.*?';
+$string = 'You deleted C:\\\\*.*?';
```

Comment:
How about `\\\'`?

Reply:
Ya im confused - the test is

```php
$string = 'You deleted C:\\*.*?';
$string = 'You deleted C:\*.*?';
```

So I'm not sure why these are the same (in either case)... The snapshot escaping just makes this so much more confusing haha

Reply:
and `\`, `\\`, `\\\` just for verification tests

Reply:
will do 👍

Reply:
$string = 'You deleted C:\\*.*?'; - just two slash
$string = 'You deleted C:\*.*?'; - just one slash
In snapshot each slash should be escaped

Reply:
err actually `\` and `\\\` are invalid because they escape the closing quote

Reply:
@mgrip when I think that something is wrong or wrong, i just create `test.php` with content and `echo $string;` and look how it is output :smile:

Reply:
right right - so snapshot should be
```php
$string = 'You deleted C:\\\\*.*?';
$string = 'You deleted C:\\*.*?';
```
right?

I'm just not sure why its incorrect atm. They're both the same

Reply:
@mgrip this examples was got from `php.net` :smile:

Reply:
yep it is right, first with two slashes, second with one slash

Reply:
Ok its a parser error - I'll open a new issue

![image](https://user-images.githubusercontent.com/13454680/37164776-42fb3304-22c9-11e8-8d35-31575eaebcc0.png)